rust/parser/typeql.pest

```diff
@@ -171,21 +171,24 @@ struct_value = { expression_value | expression_struct }
 
 // FETCH QUERY =================================================================
 
-clause_fetch = { FETCH ~ projection_object }
+clause_fetch = { FETCH ~ fetch_object }
 
-projection = { projection_single | projection_object | projection_list }
+fetch_some = { fetch_single | fetch_object | fetch_list }
 
-projection_object = { CURLY_OPEN ~ projection_object_field ~ ( COMMA ~ projection_object_field )* ~ CURLY_CLOSE }
-projection_object_field = { projection_key ~ COLON ~ projection }
-projection_key = { quoted_string_literal }
+fetch_object = { CURLY_OPEN ~ fetch_body  ~ CURLY_CLOSE }
+fetch_body = { fetch_object_entries | fetch_attributes_all }
+fetch_object_entries = { fetch_object_entry ~ ( COMMA ~ fetch_object_entry )* }
+fetch_object_entry = { fetch_key ~ COLON ~ fetch_some }
+fetch_key = { quoted_string_literal }
```

Comment:
Removed the word `projection` as well, since these are no longer really 'projecting' one thing into another space.